### ISSUE TITLE
If seqwish supports output with nodes as single characters?

### ISSUE THREAD

**Issue by CuteYisin** [ekg/seqwish#120]
Hi there,

I am using seqwish to build pangenome graph. I would like to inquire if seqwish supports output with nodes as single characters. Could you please confirm if this functionality is available?

Thank you!

**Comment by ekg** [comment]
Yes, but you will want to use odgi chop -c 1 to make the graph have this format.

**Comment by CuteYisin** [comment]
Thank you for the great response! Your answer was really helpful.
